drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw/host_rpu_umac_if.h

```diff
@@ -2495,6 +2495,17 @@ struct nrf_wifi_umac_cmd_get_tx_power {
 	struct nrf_wifi_umac_hdr umac_hdr;
 } __NRF_WIFI_PKD;
 
+/**
+ * struct nrf_wifi_umac_cmd_get_reg - get regulatory.
+ * @umac_hdr: UMAC command header. Refer &struct nrf_wifi_umac_hdr.
+ *
+ * This structure represents the command to get regulatory domain.
+ */
+
+struct nrf_wifi_umac_cmd_get_reg {
+	struct nrf_wifi_umac_hdr umac_hdr;
```

Comment:
do we need need an empty structu? Can't we use just usee the `nrf_wifi_umac_hdr` ?

Reply:
Yes we can use. We have other command structures with only "umac_hdr", to align with them used above command.
we can further cleanup this by replacing all the empty structures with "umac_hdr", but that will show lot of differences from the interface file from UMAC side which went in to ROM.
I prefer to take up this change in vega.